pkg/test/templateupdaterequest/templateupdaterequest.go

```diff
@@ -0,0 +1,84 @@
+package templateupdaterequest
```

Comment:
I'm curious - why do we have these functions/file here and not in host-operator repo?
I'm asking since I expect that the function should be used only in host-operator - having it here makes any future modification more difficult.

Reply:
because we have similar funcs for MasterUserRecord. But yes, it's harder to maintain.

Reply:
The MUR and UserAccount functions/assertions are here because they are used in both repos host-operator and member-operator. The rest of such functions stays in the repos where they are used. 
I don't really see any benefit in having it here.

Reply:
sigh... ok, I'll move this package in the host-operator then 😬

Reply:
done in 100644